packages/block-library/src/embed/index.js

```diff
@@ -15,10 +15,14 @@ export const name = 'core/embed';
 
 export const settings = getEmbedBlockSettings( {
 	title: _x( 'Embed', 'block title' ),
-	description: __( 'The Embed block allows you to easily add videos, images, tweets, audio, and other content to your post or page.' ),
+
+	description: __( 'Embed videos, images, tweets, audio, and other content from external sources.' ),
```

Comment:
I think this should say "posts" rather than "tweets" to be more general and not reference any specific platform or service.